### PR TITLE
V1

### PR DESCRIPTION
**Pull request by fabiooshiro**

### CODE REVIEW

File: internal/model/report_repository.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func (r *ReportRepository) CreateTables() error {
 	if err == nil {
 		slog.Debug("Reports table created")
 	} else {
-		slog.Error("Create table error", slog.String("Error", err.Error()))
+		slog.Error("Create table error", "error", err)
```
Comment (fabiooshiro) [inline internal/model/report_repository.go:29]: image

Comment (fabiooshiro) [reply internal/model/report_repository.go:29]: just to keep the simplicity of the original format
image

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -30,7 +26,7 @@ jobs:
     - name: Lint
       uses: golangci/golangci-lint-action@v5
```
Comment (fabiooshiro) [inline .github/workflows/ci.yaml:27]: image
25 vs 17k linhas do antigo

File: internal/model/report_repository.go (hunk 1 above)
Comment (sandhilt) [reply internal/model/report_repository.go:29]: golangci-lint have problems with this and CI break.

Comment (sandhilt) [reply internal/model/report_repository.go:29]: For future reference:
image

Comment (fabiooshiro) [reply internal/model/report_repository.go:29]: the lint is not complaining anymore.
You just need to put a label before the object.
slog.Error("message", "label", value)

Comment (sandhilt) [reply internal/model/report_repository.go:29]: Maybe don't work, you can see what he see here, only `string` or `slog.Attr` and maybe `Interface`. Other types can don't work.

https://github.com/golang/tools/blob/master/go/analysis/passes/slog/slog.go#L115-L121

**Review by sandhilt — APPROVED** [review]
LGTM

**Pull request merged**